src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -1039,25 +1039,55 @@ There are various error codes for different error scenarios. The following table
+
+When validating required properties, we are usually talking about references to other data or metadata. In these cases there are 3 main criteria:
+1. The reference is present, and not null, in the payload.
+2. The reference points to the right type of data and exists in the database
+3. The user have access to see the reference
+
+If the first condition fails, the import will fail with a message about a missing reference. However, if the reference points to something that doesnt exists, or which the user dont have access to, both cases will result in a message about the reference not being found.
+
+#### Formats
+Some of the properties of tracker objects require a specific format. When importing data, each of these properties are validated against the expected format, and will return different errors depending on which property has a bad format. Some examples of properties that are validated this way:
+- UIDs (These cover all references to other data or metadata in DHIS2.)
+- Dates
+- Geometry (The coordinates must match the format as specified by it's type)
+
+#### User access
+All data imported will be validated based on the metadata of the data imported (Sharing) and the organisation unit of the data imported (Organisation Unit Scopes). More information about sharing and organisation unit scopes can be found in the following sections.
+
+Sharing is validated at the same time as references are looked up. That means a references the user cannot access, will be treated as if it doesn't exist. Sharing is checked for each metadata referenced in the data being imported.
+
+Organisation units on the other hand, serve a dual purpose. Primarily, it will make sure that data can only be imported when it's imported for an organisation unit the user have within their "capture scope". Secondly, organisation units are also used to restrict what programs are available. That means if you are trying to import data for an organisation unit which does not have access to the program you are importing, the import will be invalid.
+
+User's with the `ALL` authority transcends, in these cases, validations related to the user, but not programs.
+
+#### Attribute and Data values
+
+Attributes and data values are part of a tracked entity and an event respectively. However, attributes can be linked to a tracked entity either through it's type (TrackedEntityType) or it's program (Program). Additionally, attributes can also be unique.
+
+The initial validation done in the import, is to make sure the value provided for an attribute or data element conforms to the type of value expected. For example, if you import a value for a data element with a numeric type, the value is expected to be numeric. Any errors related to a mismatch between a type and a value will result in the same error code, but with a specific message related to the type of violation.
```

Comment:
Would be good to mention that validation also depends on the validationStrategy set on program? ON_COMPLETE, ON_UPDATE_AND_INSERT and etc.

Reply:
Great feedback! This is the kind of details I might overlook. Will add info